frontend/src/pictureBookView/pictureBookView.scss

```diff
@@ -0,0 +1,36 @@
+.submit-btn {
+    background-color: #278E73;
+    color: white;
+    margin-bottom: 3vh;
+    margin-top: 10vh;
+}
+
+.box {
+    background: #5941A9;
+    border-radius: 1vh;
+    margin: 5vh 1vw 3vh 1vw;
+    padding: 2vh;
+    width: 100%;
+}
+
+.modal-show {
+    translateY: 0;
+    opacity: 1;
+}
+
+.modal-hide {
+    translateY: -100vh;
```

Comment:
It looks to me like these `translateY` properties should be `transform: translateY(value)`. I wouldn't expect just supplying the transform value `translateY` as the property key to work, unless there's some strange webpack conversion happening.